pkg/registry/pod/registry.go

```diff
@@ -39,3 +39,61 @@ type Registry interface {
 	// Delete an existing pod
 	DeletePod(ctx api.Context, podID string) error
 }
+
+// Storage is an interface for a standard REST Storage backend
+// TODO: move me somewhere common
```

Comment:
Yeah, same reasoning.  I was thinking api/rest, but it might be even more fundamental 


> On Feb 13, 2015, at 6:46 PM, Brian Grant <notifications@github.com> wrote:
> 
> In pkg/registry/pod/registry.go:
> 
> > @@ -39,3 +39,61 @@ type Registry interface {
> >  	// Delete an existing pod
> >  	DeletePod(ctx api.Context, podID string) error
> >  }
> > +
> > +// Storage is an interface for a standard REST Storage backend
> > +// TODO: move me somewhere common
> Why not pkg/apiserver/interfaces.go?
> 
> However, it seems to me that referencing apiserver here violates our layering.
> 
> I'm ok with this as an intermediate step, but we should draw a (ideally acyclic) diagram of how we want the code to be organized.
> 
> —
> Reply to this email directly or view it on GitHub.
>